.github/workflows/r-cmd-check.yml

```diff
@@ -34,8 +34,8 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: ubuntu-latest, r: '3.6', repos: 'https://cran.microsoft.com/snapshot/2020-02-29/'}
           - {os: ubuntu-latest, r: '4.0', repos: 'https://cran.microsoft.com/snapshot/2021-03-31/'}
+          - {os: ubuntu-latest, r: '4.1', repos: 'https://cran.microsoft.com/snapshot/2021-03-31/'}
```

Comment:
Please use `2022-03-10` as the snapshot date which is the release date of R v4.1.3.